lib/diagnostic.rb

```diff
@@ -21,20 +27,26 @@ def sum(enum)
   # in the numbers enumerable are odd
   def some_odd(numbers)
     # Your code goes here
+   numbers.one? { |num| num.odd? }
```

Comment:
This will return `true` if _exactly one_ of the entries is odd. What if there were two?
The right tool to use in this scenario is `.any?`